src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocmethod/InputMissingJavadocMethodPublicOnly.java

```diff
@@ -87,16 +86,16 @@ public void method(StringBuffer aA)
 
 
     /**
-       A param tag should not be required here when relaxed about Javadoc.
-       Writing a little documentation should not be worse than not
-       writing any documentation at all.
+     * A param tag should not be required here when relaxed about Javadoc.
+     * Writing a little documentation should not be worse than not
+     * writing any documentation at all.
```

Comment:
I would probably revert any formatting changes or put them in a separate commit.

Reply:
Moved reformatting to new commit.

Reply:
Resolved.